rskj-core/src/main/java/org/ethereum/core/BlockHeader.java

```diff
@@ -96,11 +97,14 @@
     private byte[] minimumGasPrice;
     private int uncleCount;
 
-    public BlockHeader(byte[] encoded) {
-        this((RLPList) RLP.decode2(encoded).get(0));
+    /* Indicates if this block header cannot be changed */
+    private volatile boolean sealed;
+
+    public BlockHeader(byte[] encoded, boolean sealed) {
```

Comment:
I'd vote for a method that only receives the rlp data and defaults sealed to true.

```
public BlockHeader(byte[] encoded) {
    this(encoded, true);
}
```